src/main/webapp/i18n/en/instructorDashboard.json

```diff
@@ -13,6 +13,11 @@
         "archive": {
             "title": "Archive",
             "question": "Are you sure you want to archive ALL student repositories for the programming exercise {{ exerciseTitle }} in the course {{courseTitle}}?"
+        },
+        "exportRepos": {
+            "warning": "Confirm Export",
+            "title": "Download Repos",
+            "question": "Enter all ids of students you want to download the repo from"
```

Comment:
Can you write "logins" instead of "ids" and give a simple example for TUM students, e.g. 'ga87fix, ne23kow'